test/libsolidity/semanticTests/abiEncoderV1/abi_encode_empty_string.sol

```diff
@@ -1,3 +1,4 @@
+pragma abicoder v1;
```

Comment:
These files have the `ABIEncoderV1Only` setting, but adding the pragma for explicitness. Probably this is a subjective choice, but it helps to see the difference easily when there are files with the `_v1` and `_v2` suffix.

Reply:
I agree. I think that all files that require v1 should have the v1 pragma.